From 54e3197024b8fb3c8d92a2d84e255030a68ae250 Mon Sep 17 00:00:00 2001
From: Santhosh Kumar <santhosh.kumar.selvaraj@qt.io>
Date: Mon, 22 Sep 2025 15:55:50 +0200
Subject: [PATCH 063/239] Mark security header for the qt labs synchronizer

QUIP: 23
Task-number: QTBUG-136184
Change-Id: I27e9fc93ca4503459263aa4086d89c79fc1bebfa
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
(cherry picked from commit ab008ba17bb60de807e47a469da47fe63de8d0fd)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/labs/synchronizer/qqmlsynchronizer.cpp | 1 +
 src/labs/synchronizer/qqmlsynchronizer_p.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/labs/synchronizer/qqmlsynchronizer.cpp b/src/labs/synchronizer/qqmlsynchronizer.cpp
index 4b125a2bd8..d8246705ba 100644
--- a/src/labs/synchronizer/qqmlsynchronizer.cpp
+++ b/src/labs/synchronizer/qqmlsynchronizer.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qqmlsynchronizer_p.h"
 
diff --git a/src/labs/synchronizer/qqmlsynchronizer_p.h b/src/labs/synchronizer/qqmlsynchronizer_p.h
index ac90bd9efd..cf2686145b 100644
--- a/src/labs/synchronizer/qqmlsynchronizer_p.h
+++ b/src/labs/synchronizer/qqmlsynchronizer_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQMLSYNCHRONIZER_P_H
 #define QQMLSYNCHRONIZER_P_H
-- 
2.51.2

